src/main/java/bob/task/ToDos.java

```diff
@@ -0,0 +1,41 @@
+package bob.task;
+
+import bob.exception.InvalidInputException;
+
+/**
+ * Tasks without any date/time attached to it.
+ */
+public class ToDos extends Task {
```

Comment:
```suggestion
public class Todo extends Task {
```
Perhaps it would be good to rename it to either Todo or ToDo? (Non plural form)